### DISCUSSION TITLE
Reviving players doesnt work. Also /fakedeath doesnt work, /close and /open doesnt work, and /bomb/bombard are very spotty on working

### DISCUSSION THREAD

**Discussion by Lchaxlover** [winstxnhdw/lc-hax#492] [Q&A]
With the /heal player command it says heal or revive a player but reviving them never works.

**Comment by Lchaxlover** [comment]
Also /fakedeath doesnt work, /close and /open doesnt work, and /bomb/bombard are very spotty on working

**Comment by winstxnhdw** [comment]
`heal` can't revive other players, only yourself. I haven't tested `open` or `close` in a while but the rest should work as per normal. You should test on vanilla.
